libr/anal/p/anal_ppc_gnu.c

```diff
@@ -9,9 +9,11 @@
 // XXX addr should be off_t for 64 love
 static int ppc_op(RAnal *anal, RAnalOp *op, ut64 addr, const ut8 *_bytes, int len) {
 //int arch_ppc_op(ut64 addr, const u8 *bytes, struct op_t *op)
-// TODO swap endian here??
-	char bytes[1024];
-	r_mem_copyendian ((ut8*)bytes, _bytes, 4, 1);
+	ut8 bytes[1024];
+	if (anal->swap_endian)
```

Comment:
missing spaces and use brackets. in fact r_mem_copyendian was already handling the endian properly, no need to reimplement it here